debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/RecordMakers.java

```diff
@@ -201,37 +203,20 @@ protected int createRecords(Struct source, Map<String, ?> offset, Operation oper
             return 1;
         }
 
-        protected String objectIdLiteralFrom(Document obj) {
-            if (obj == null) {
+        protected String idObjToJson(Object idObj) {
+            if(idObj == null) {
```

Comment:
nit: does this pass checkstyle? Should have space after `if`.

Reply:
unsure to be honest... but probably not. the build process itself doesn't seem to trigger checkstyle, does it?

Reply:
w.r.t JavaDoc: the (internal) protected methods I changed didn't have any kind of docs before. 

the key struct format changes with the json-serialized id field as string and the renaming to "id" to be consistent with the other connectors is IMHO something that needs to be documented for the release notes and the official docs page of the connector. don't see any real benefit to have this in the javadoc as this wasn't explained for the previous version either within the code?

but if you tell me what of the small parts I've done should get additional docs I'd be happy to provide some of course :)

Reply:
@hpgrahsl In terms of documentation, indeed the [MongoDB connector docs](http://debezium.io/docs/connectors/mongodb/) published on the website will need some update. They should be updated to show the new field name and also we should spend some words on the id format, e.g. when it comes to numeric types, object ids or complex key types. The source of the documentation is [here](https://github.com/debezium/debezium.github.io/blob/develop/docs/connectors/mongodb.asciidoc); perhaps you could send a PR for updating them?

Reply:
We'll also mention the changed behavior in the release notes of course.

Reply:
THX for the pointer to the docs. I'll try to describe to most relevant changes. Would you prefer a separate PR or should I do this within this context?

Reply:
@gunnarmorling BTW can you please comment on the checkstyle issue that @rhauch brought up? I couldn't spot any violations during the build process but maybe something is configured wrong in my environment?

Reply:
CheckStyle is executed by default, but our [rules file](https://github.com/debezium/debezium/blob/master/support/checkstyle/src/main/resources/checkstyle.xml) does currently not contain a check for whitespaces after `if` literals.

I suggest you change the incriminating location as part of this PR and then we handle it separately to add this check to our rules file and adjust the code base globally as needed. That's what we'd have to add under the `TreeWalker` module:

    <module name="WhitespaceAfter"/>

Reply:
I've filed https://issues.jboss.org/browse/DBZ-362 for taking care of that.

Reply:
@gunnarmorling thx for the clarification. I'll checkt and fix the violations manually and will update the PR accordingly together with the other change.